benchmarks/java/pom.xml

```diff
@@ -0,0 +1,148 @@
+
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2011 Google Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Remove this copy right notice. It's incorrect. We don't use Apache License.

Reply:
Done